dev/ci/README-developers.md

```diff
@@ -0,0 +1,165 @@
+Information for developers about the CI system
+----------------------------------------------
+
+When you submit a pull request (PR) on Coq GitHub repository, this will
```

Comment:
```suggestion
When you submit a pull request (PR) on the Coq GitHub repository, this will
```

or

```suggestion
When you submit a pull request (PR) on Coq's GitHub repository, this will
```